src/platform/Darwin/Logging.cpp

```diff
@@ -29,7 +29,7 @@ void LogV(const char * module, uint8_t category, const char * msg, va_list v)
     pthread_threadid_np(NULL, &ktid);
 
     char formattedMsg[CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE];
-    int32_t prefixLen   = snprintf(formattedMsg, sizeof(formattedMsg), "[%ld] [0x%" PRIx64 "] CHIP: [%s] ", ms, ktid, module);
+    int32_t prefixLen   = snprintf(formattedMsg, sizeof(formattedMsg), "[%ld] [0x%" PRIx32 ":0x%" PRIx64 "] CHIP: [%s] ", ms, getpid(), ktid, module);
```

Comment:
It might be better to:

1. Cast the return type of `getpid()` to `long long` (to avoid problems with trying to guess which integer type `pid_t` is in practice).
2. Use `%lld` for the fomat for the PID; PIDs are usually not shown in hex and using decimal would make it easier to correlate with other things (e.g. ps output).